contracts/test/ParityTest.t.sol

```diff
@@ -36,9 +36,11 @@ import { AddressRegistry } from "../AddressRegistry.sol";
 
 contract ParityTest is AddressRegistry, StateManipulations, TestUtils {
 
-    uint256 constant WAD = 10 ** 18;  // ETH  precision
-    uint256 constant BTC = 10 ** 8;   // WBTC precision
-    uint256 constant USD = 10 ** 6;   // USDC precision
+    uint256 constant WAD      = 10 ** 18;  // ETH  precision
+    uint256 constant BTC      = 10 ** 8;   // WBTC precision
+    uint256 constant USD      = 10 ** 6;   // USDC precision
+    uint256 constant ZERO     = uint256(0);
```

Comment:
Was this needed?

Reply:
It was an attempt to remove items from the stack, but I didn't help. I do think is nice though